traits/src/float.rs

```diff
@@ -323,8 +323,8 @@ pub trait Float
     /// ```
     fn signum(self) -> Self;
 
-    /// Returns `true` if `self` is positive, including `+0.0` and
-    /// `Float::infinity()`.
+    /// Returns `true` if `self` is positive, including `+0.0`,
+    /// `Float::infinity()`, and `f64::NAN` with newer versions of Rust.
```

Comment:
Any ideas on how to make this less confusing? Right now it's not clear what "newer versions of Rust" applies to.